packages/web-components/src/components/button/button.stories.ts

```diff
@@ -372,11 +377,12 @@ export const SetOfButtons = {
     isSelected,
     linkRole,
     size,
+    stacked,
     tooltipAlignment,
     tooltipPosition,
     type,
   }) =>
-    html` <cds-button-set>
+    html` <cds-button-set .stacked="${stacked}">
```

Comment:
```suggestion
    html` <cds-button-set stacked="${stacked}">
```
i don't think the . is necessary here

Reply:
Ordinarily not, but that will get pulled into the `Canvas` source panel as an unrendered prop and it might be confusing to people.